chrome/content/zotero/overlay.js

```diff
@@ -188,7 +189,7 @@ var ZoteroOverlay = new function()
 		Zotero.debug("zoteroPane.getAttribute('minheight'): " + zoteroPane.getAttribute('minheight'));
 		Zotero.debug("savedHeight: " + savedHeight);
 		*/
-		
+
```

Comment:
The note is because PRs generally shouldn't change unrelated lines, even for stylistic reasons — you can use `git checkout -p` (or `git reset -p` to keep) to get rid of any lines in the diff that don't belong.

In terms of whitespace, file size isn't a concern. We're not religious about this in submitted code, but we generally leave whitespace-line indents in place. In my opinion it makes for easier editing — this way if you go to add a line between two indented blocks, you're already at the right indent level without having to hit Tab or go up to the end of the block above.